terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1497,6 +1500,42 @@ def qq(*words):
                 'depends_on': ['aws_iam_service_linked_role.gitlab_ssm']
             }
         },
+        'aws_cloudformation_stack': {
```

Comment:
Let's just move this to the shared component and add a FIXME to `aws_inspector2_enabler` above to move that one later.

Reply:
https://github.com/DataBiosphere/azul/issues/5760